tests/jenkins/TestDockerPromoteWithEcrAndDockerhubJob.groovy

```diff
@@ -28,6 +28,7 @@ class TestDockerPromoteWithEcrAndDockerhubJob extends BuildPipelineTest {
         binding.setVariable('IMAGE_TAG', imageTag)
         binding.setVariable('AWS_ACCOUNT_ARTIFACT', accountName)
         binding.setVariable('PLATFORM', 'docker-hub + ECR')
+        binding.setVariable('TAG_LATEST', 'false')
```

Comment:
Since latest=false, the txt file for this test needs to be updated as well right?

I only see tests regression files for `latest=true`

Reply:
No. Since the latest = false, there will be no change to these txt files, since the flow would remain the same. The flow only changes if latest=true